File: Beecrowd/C/1158 Sum of Consecutive Odd Numbers III.c
#include <stdio.h>

int main()
{
    int t,i,j,sum=0,X,Y;
    
    scanf("%d",&t);
    for(i=0;i<t;i++){
        
        scanf("%d %d",&X,&Y);
        
        if(X %2==0){
            X+=1;
        }
    
    for (j=1;j<Y+1;j++){
         sum+=X;
         X+=2 ;
    }
     printf("%d\n",sum);

    sum=0;
    }
    
    
   return 0;
}

File: Beecrowd/Python/1158 Sum of Consecutive Odd Numbers III.py
for _ in range(int(input())):
    sum = 0 
    X,Y = map(int,input().split());

    if(X %2==0):
        X+=1

    
    for i  in range(1,Y+1):
        sum+=X;
        X+=2 ;
    print(sum)
    
    X=0;
    Y=0;
    sum=0;
